testing/src/main/java/org/camunda/commons/testing/testcontainers/TestcontainersHelper.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.commons.testing.testcontainers;
+
+import org.testcontainers.shaded.org.apache.commons.lang.StringUtils;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+public class TestcontainersHelper {
+
+  public static String resolveImageName(String imageProperty, String defaultImage) {
+    String image = TestcontainersConfiguration.getInstance().getEnvVarOrProperty(imageProperty, defaultImage);
+    if (image == null) {
+      throw new IllegalStateException("An image to use for Oracle containers must be configured. " +
```

Comment:
This exception might not only occur for **Oracle** containers, right?

Reply:
Yes, this was with the initial implementation. I will rephrase it to be more general so that it applies to all containers.